frontend/src/components/SupportedAppTitle.tsx

```diff
@@ -13,9 +13,10 @@ const SupportedAppTitle: React.FC<SupportedAppTitleProps> = ({ odhApp, showProvi
   const title = odhApp.spec.displayName;
   const icon = (
     <span style={{ whiteSpace: 'nowrap' }}>
-      <Tooltip content={`${ODH_PRODUCT_NAME} certified and supported`}>
+      <Tooltip content={`${ODH_PRODUCT_NAME} certified and supported`} data-testid="tool-tip">
         <Button variant="plain" style={{ padding: 0, verticalAlign: 'middle' }}>
           <img
+            data-testid={`${ODH_PRODUCT_NAME}`}
```

Comment:
Not everything requires a test ID. In this example you can easily find the single image nested within the `tool-tip` element.
Furthermore always assign static values to test ids.